### PR TITLE
reduce build time: Initial small wins

### PR DESCRIPTION
**Pull request by llogiq**
I also fixed the benchmarks en passant; I noticed they were broken for a while.

### CODE REVIEW

**Review by ggordonhall — APPROVED** [review]

File: server/bleep/src/indexes/file.rs [hunk 1]
```diff
@@ -167,7 +167,7 @@ impl Indexable for File {
         repo: &Repository,
         repo_info: &RepoHeadInfo,
         writer: &IndexWriter,
-    ) -> Result<()> {
+    ) -> Result<()> {
```
Comment (rsdy) [inline server/bleep/src/indexes/file.rs:170]: Empty spaces at the end of line!

**Review by rsdy — APPROVED** [review]

File: server/bleep/src/indexes/file.rs (hunk 1 above)
Comment (ggordonhall) [reply server/bleep/src/indexes/file.rs:170]: We need to get the CI back up. Have to configure qdrant builds: https://github.com/BloopAI/bloop/actions/runs/3902002219/jobs/6664534799

Comment (llogiq) [reply server/bleep/src/indexes/file.rs]: Oops, fixed.

**Comment by llogiq** [comment]
rebased.

**Comment by llogiq** [comment]
Didn't we have git-lfs before?

**Comment by llogiq** [comment]
Now it appears we are throttled again, this time downloading onnxruntime. We should probably cache it on the CI image.

**Pull request merged**